routes/api/index.js

```diff
@@ -10,8 +10,16 @@ router.get('/board', (req, res) => {
 
 // test route
 router.post('/board', (req, res) => {
-	const foo = req.body.body
-	return res.json({ foo })
+	const foo = req.body.body;
+	return res.json({ foo });
+});
+
+// move made by user are sent here, verified and updated
+router.post('/move', (req, res) => {
+	console.log(req.body, "here's your data");
+	// Verify move
+	// const newBoard = createNewGame(req.body)
+	// res.render('board', newBoard)
```

Comment:
I'm working on this in parallel. I'd suggest this until I get around to actually implementing the move. This way, no matter what move you make, the default board is returned, so you can think about re-rendering and stuff :-)
```suggestion
	return res.json(BoardFactory.toObject(new Board());
```